appside/frames/likert.js

```diff
@@ -0,0 +1,122 @@
+"use strict";
+
+/**
+ * Rendering (View) code for likert frame
+ * @author Rachel Sitt
+ */
+
+
+/**
+ * A frame composed of a list of things.
+ * 
+ * Things in the list are to be rendered as a list of radio buttons.
+ **/
+class LikertFrame extends Frame {
+
+    /**
+     * Construct LikertFrame from an object
+     * 
+     * @param frame_data -- Object containing the frame's data. Expected fields:
+     *    frame_data.title (string) -- frame's title
+     *    frame_data.instructions (string) -- frame's instructions for user
+     *    frame_data.question (string) -- Text to appear before the list of statements
```

Comment:
`frame_data.question` is outdated here.

`frame_data.question**s**` should be listed here, and should be specified here as a list of question/response pairs, where the question is a string and the response is undef or an integer within the correct range.